src/gp_config.c

```diff
@@ -538,6 +538,17 @@ static int load_services(struct gp_config *cfg, struct gp_ini_context *ctx)
                     goto done;
                 }
             }
+
+            cfg->svcs[n]->min_lifetime = 0;
+            ret = gp_config_get_int(ctx, secname, "min_lifetime", &valnum);
+            if (ret == 0) {
+                if (valnum >= 0) {
```

Comment:
set unconditionally, someone may be resetting a default to 0

Reply:
I'm not following.  This doesn't prevent the user from explicitly setting the value to zero.